src/js/views/admin/sidebar/clinician/clinician-sidebar_views.js

```diff
@@ -0,0 +1,149 @@
+  },
+  templateContext() {
+    return {
+      isNew: this.model.isNew(),
+    };
+  },
+  onDomRefresh() {
+    if (this.model.isNew()) {
+      this.ui.input.focus();
+    }
+  },
+});
+
+const EmailView = View.extend({
+  className: 'pos--relative',
+  template: hbs`<input class="input-primary w-100 js-input" placeholder="{{ @intl.admin.sidebar.clinician.clinicianSidebarViews.emailView.placeholder }}" value="{{ email }}" />`,
+  behaviors: [InputWatcherBehavior],
+  ui: {
+    input: '.js-input',
+  },
+  onWatchKeydown(evt) {
+    if (evt.which === _.ENTER_KEY) {
+      evt.preventDefault();
+      return;
+    }
+  },
+  onWatchChange(text) {
+    this.model.set('email', text);
+  },
+  templateContext() {
+    return {
+      isNew: this.model.isNew(),
+    };
+  },
+});
+
+const SaveView = View.extend({
+  className: 'u-margin--t-8 u-text-align--right',
+  template: hbs`
+    <button class="button--text u-margin--r-4 js-cancel">{{ @intl.admin.sidebar.clinician.clinicianSidebarViews.saveView.cancelBtn }}</button>
+    <button class="button--green js-save">{{ @intl.admin.sidebar.clinician.clinicianSidebarViews.saveView.saveBtn }}</button>
+  `,
+  triggers: {
+    'click .js-cancel': 'cancel',
+    'click .js-save': 'save',
+  },
+});
+
+const DisabledSaveView = View.extend({
+  className: 'u-margin--t-8 u-text-align--right',
+  template: hbs`<button class="button--green" disabled>{{ @intl.admin.sidebar.clinician.clinicianSidebarViews.disabledSaveView.saveBtn }}</button>`,
+});
+
+const SidebarView = View.extend({
+  className: 'sidebar flex-region',
+  template: ClinicianSidebarTemplate,
+  triggers: {
+    'click .js-close': 'close',
+    // 'click .js-menu': 'click:menu',
+  },
+  childViewTriggers: {
+    'save': 'save',
+    'cancel': 'cancel',
+  },
+  regions: {
+    name: '[data-name-region]',
+    email: '[data-email-region]',
+    save: '[data-save-region]',
+    role: '[data-role-region]',
+    // access: '[data-access-region]',
+    // groups: '[data-groups-region]',
+  },
+  onRender() {
+    this.showForm();
+    this.showRole();
+  },
+  showName() {
+    this.showChildView('name', new NameView({ model: this.model }));
+  },
+  showEmail() {
+    this.showChildView('email', new EmailView({ model: this.model }));
+  },
+  showRole() {
+    const isDisabled = this.model.isNew();
+    const roleComponent = new RoleComponent({ model: this.model, state: { isDisabled } });
+
+    this.listenTo(roleComponent, 'change:role', role => {
+      this.model.saveRole(role);
+    });
+
+    this.showChildView('role', roleComponent);
+  },
+  showSave() {
+    if (!this.model.isNew()) return;
+    if (!this.model.isValid()) return this.showDisabledSave();
+
+    this.showChildView('save', new SaveView({ model: this.model }));
+  },
+  showDisabledSave() {
+    this.showChildView('save', new DisabledSaveView());
+  },
+  showForm() {
+    this.listenTo(this.model, 'change:name change:email', this.showSave);
```

Comment:
rather than bailing out of `showSave` if `!isNew` this listener could just be in the `isNew` below.